examples/draw_and_compare_graphs_with_same_layout.py

```diff
@@ -0,0 +1,51 @@
+"""
+.. _ex-draw-graphs:
+
+=============================================================
+Drawing graphs and setting their layout for visual comparison
+=============================================================
+
+One can draw a graph without setting the `pos` argument, in that case graphviz will choose how to place the nodes.
+See https://graphviz.readthedocs.io/en/stable/api.html?highlight=render#graphviz.Digraph.render
```

Comment:
```suggestion
See https://graphviz.readthedocs.io/en/stable/api.html?highlight=render#graphviz.Digraph.render.
In this example, we demonstrate how to visualize various different graphs with a fixed layout
for the nodes, so that they are easily comparable.
```